draft-ietf-rtcweb-mdns-ice-candidates.md

```diff
@@ -318,9 +318,18 @@ browsing context.
 Network Interface Enumeration
 -----------------------------
 
-Even when local IP addresses are not exposed, the number of mDNS hostname candidates can still provide a fingerprinting dimension.
-This is in particular the case for network interfaces with limited connectivity that will not generate any server-reflexive or relay candidate.
-No new information is provided when restricting mDNS hostname candidates to default route only.
+Even when local IP addresses are not exposed, the number of mDNS hostname
+candidates can still provide a fingerprinting dimension. This is in particular
+the case for network interfaces with limited connectivity that will not generate
+any server-reflexive or relay candidate.
+
+Ideally, the number of discoverable server-reflexive candidates will be equal
```

Comment:
Not sure I follow; how is this possible? Wouldn't we always have equal or fewer srflx candidates?

Reply:
Relative to host candidates, yes, but @youennf wants to limit the derived mDNS candidates. If we expose more mDNS candidates than srflx candidates, we're leaking that there are (private) host candidates in addition to the ones used to create the srflx ones, giving away a little bit of private information?

Reply:
Ah, I see what you're getting at. Maybe we should leave this for later, as I can think of a couple issues that this could cause:
- no candidates at all if a STUN server isn't provided or isn't reachable
- host candidates aren't exposed until the STUN check comes back, increasing latency

Reply:
Right, the idea was to describe in terms of total number of "potential" server-reflexive candidates, not in terms of implementation strategy. Maybe this criteria is not great.
I'll do another try at it.